readthedocs/proxito/views/serve.py

```diff
@@ -176,16 +176,25 @@ def get(self,
             )
             raise Http404('Invalid URL for project with versions')
 
-        # TODO: un-comment when ready to perform redirect here
-        # redirect_path, http_status = self.get_redirect(
-        #     final_project,
-        #     lang_slug,
-        #     version_slug,
-        #     filename,
-        #     request.path,
-        # )
-        # if redirect_path and http_status:
-        #     return self.get_redirect_response(request, redirect_path, http_status)
+        redirect_path, http_status = self.get_redirect(
```

Comment:
We could just call `final_project.get_redirect_path_with_status()` directly, we aren't adding anything extra on `get_redirect`.